hazelcast/src/test/java/com/hazelcast/client/impl/spi/impl/ClientInvocationFutureTest.java

```diff
@@ -278,29 +277,25 @@ public void test_handle() throws Exception {
     public void test_acceptEither() throws Exception {
         CompletableFuture nextStage = invocationFuture.acceptEither(newCompletedFuture(null),
                 t -> ignore(null));
-        invocationFuture.complete(response);
 
         assertEquals(null, nextStage.get(10, TimeUnit.SECONDS));
         verify(callIdSequence).forceNext();
-        verify(callIdSequence, times(2)).complete();
+        verify(callIdSequence, times(1)).complete();
     }
 
     @Test
     public void test_applyEither() throws Exception {
-        CompletableFuture nextStage = invocationFuture.applyToEither(newCompletedFuture(null),
-                (t) -> t);
-        invocationFuture.complete(response);
```

Comment:
Completing the `invocationFuture` is both unnecessary (because it is sufficient for one `CompletableFuture` to be completed for `*Either` methods to be executed) and using a value other than `null` may result in `(t) -> t` function to return that value as there is no guarantee which `CompletableFuture`'s completion value will be passed in the `Function`.